can/interfaces/socketcan/socketcan_ctypes.py

```diff
@@ -125,12 +125,16 @@ def recv(self, timeout=None):
 
     def send(self, msg, timeout=None):
         frame = _build_can_frame(msg)
+
         if timeout:
             # Wait for write availability. write will fail below on timeout
-            select.select([], [self.socket], [], timeout)
+            _, ready_send_sockets, _ = select.select([], [self.socket], [], timeout)
```

Comment:
This shouldn't be part of this PR

Reply:
Yeah right, I must've started with the wrong branch or something. I would just leave it in this time. Ok?